k8s/keycloak/keycloak.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: keycloak
+  labels:
+    project: msp-invoice
+    bi: keycloak
+    app: keycloak 
+    env: dev
+    tier: backend
+spec:
+  type: NodePort
+  ports:
+  - name: wab-client
+    port: 8094
```

Comment:
This is the port of the container. As far I can see, you don't [expose](https://github.com/Lansoftdev/msp-invoice/pull/5/files#diff-378face2040da552e85b806792918143252738c240a9df7bcf15ab960ee263c0R53) it:

```
        - name: http
          containerPort: 8095
        - name: https
          containerPort: 8443
```